docs_user/modules/con_openshift-preparation-for-block-storage-adoption.adoc

```diff
@@ -84,27 +76,19 @@ spec:
       - enabled: true
         name: iscsid.service
 ----
-+
-If you use labels to restrict the nodes where the Block Storage services run, you must use a `MachineConfigPool` to limit the effects of the
-`MachineConfig` to only the nodes where your services might run. For more information, xref:about-node-selector_planning[About node selector].
-+
-If you are using a single node deployment to test the process, replace `worker` with `master` in the `MachineConfig`.
-+
-For production deployments that use iSCSI volumes, it is recommended to configure
-multipathing.
+** If you use labels to restrict the nodes where the Block Storage services run, you must use a `MachineConfigPool` to limit the effects of the
+`MachineConfig` to only the nodes where your services might run. For more information, see link:{defaultOCPURL}/nodes/index#nodes-scheduler-node-selectors-about_nodes-scheduler-node-selectors[About node selectors].
+** If you are using a single node deployment to test the process, replace `worker` with `master` in the `MachineConfig`.
+** For production deployments that use iSCSI volumes, configure multipathing.
```

Comment:
Why is it recommended for production deployments to configure multipathing?

Reply:
Because it's more robust (tolerant to network failures) and can also provide better I/O.